charts/manila-csi-plugin/values.yaml

```diff
@@ -67,7 +67,7 @@ controllerplugin:
   provisioner:
     image:
       repository: quay.io/k8scsi/csi-provisioner
-      tag: v1.4.0
+      tag: v2.0.2
```

Comment:
I used to know some RBAC changes are needed for cinder.. not sure for manila ..

Reply:
@jichenjc There are RBAC changes regarding creating volumes from snapshots, those are in the [external-snapshotter v2 PR](https://github.com/kubernetes/cloud-provider-openstack/pull/1205) though.

Reply:
I messed up .. thanks for the comments